.changeset/rich-bags-deliver.md

```diff
@@ -20,3 +21,21 @@ button {
   }
 }
 ```
+
+Fix duplicate red orange token
```

Comment:
this can be a separate `patch` level changeset

Reply:
I had snuck the red orange fix in the status color PR and realized it wasn't mentioned in the changeset.

Reply:
Each change should be it's own changeset. In this case we have two changes, one to fix an orange value, and one to add status tokens

Reply:
Ah ha, ok I understand what you are getting at now.  The description of the change in the changeset should exist as a line item under patch.  Not that the edit to the changeset needs a changeset.

Reply:
I ended up putting the patch in a separate changeset.